casingSimulations/base.py

```diff
@@ -59,3 +60,7 @@ def save(self, filename=None, directory=None):
 
         print('Saved {}'.format(f))
 
+    def copy(self):
```

Comment:
I think defining this on the base class is good enough - it gets the appropriate class to deserialize from `self.__class__` (see: https://github.com/3ptscience/properties/blob/master/properties/base/base.py#L504 )